cypress/support/pages/admin_console/manage/clients/AuthorizationTab.ts

```diff
@@ -182,4 +182,12 @@ export default class AuthorizationTab {
     this.getResourceLink(name).should("exist");
     return this;
   }
+
+  waitForExportLoad(callback: () => void) {
+    cy.intercept(
+      "/admin/realms/master/clients/*/authz/resource-server/settings"
+    ).as("export-load");
+    callback();
+    cy.wait(["@export-load"]);
+  }
```

Comment:
I don't think we need to have a callback here since Cypress should manage the command queue for us.
```suggestion
  waitForExportLoad() {
    cy.intercept(
      "/admin/realms/master/clients/*/authz/resource-server/settings"
    ).as("export-load");
    cy.wait(["@export-load"]);
  }
```